### PR TITLE
Module 1:  Capstone

### PR DESCRIPTION
**Pull request by smunozmo**
# Documentation

[Project description Video](https://www.loom.com/share/1fb076c670ae41c5b379b69c89a6e004)
[Live Demo - GitHub Pages](https://smunozmo.github.io/Module_1_Capstone/)

## Updates:

*  Fixed links paths for GitHub pages.

## Notes:

✅  - No linters errors detected. 
✅  - Used flex-box for layout. 
✅  - Used Javascript for animations. 
✅  - Featured section created dynamically.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 > Welcome to my HTML + CSS + Javascript Capstone project, feel free to look around.
 
-- Live version: [GitHub Page](https://github.com/smunozmo/HTML-basic-JavaScript-Capstone)
+- Live version: [GitHub Page](https://smunozmo.github.io/Module_1_Capstone/)
 
-![screenshot](./)
+![screenshot](./assets/img/screenshot.png)
 
 ## Built With
```
Comment (Temmarie) [inline README.md]: - [ ] Good job setting up a descriptive and professional Readme file, You need to make a few tweaks and you'll be good to go. See my comments below on the changes to make:

- Kindly add a short description of your capstone project stating its uses and purpose.
- Add more information on how to run the project locally, keeping in mind that there might be non-technical users trying to run it. Also, add the tools and software needed to run it.
 - The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (Temmarie) [inline .vscode/settings.json]: - [ ] Kindly delete this file or add it to the `.gitignore` file. Try not to commit files that are unrelated to the project, they take up space and have no relevance to the project.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+          <a href="about.html">About</a>
+          <a href="#media">Media</a>
+          <a href="#lineup">Line up</a>
+          <a href="#buytickets">Buy tickets</a>
+          <div class="menu-close">
+            <img src="assets/img/Union.svg" alt="" />
+          </div>
+        </div>
+      </nav>
+    </section>
```
Comment (Temmarie) [inline index.html]: - [ ] Kindly use `header` to group the contents of the navigation bar(both top and bottom) instead of the `section` tag.
- [ ] Also, it'll be nice if you could use one `nav` to group both the top and bottom navigation bars and then use a `section` tag to group each individually. This just makes it more semantically correct and ordered. 
Note that, you can have as many `nav` elements as you need on your page, but you don't need to wrap every link or set of links in a `nav` - only 'major navigation' should be in a `nav` element.  
- Kindly increase the font size and weight of the nav-items and make the first link(About) the same colour as is in the Behance design.
- Kindly remove the box-shadow you have on the lower navigation bar. Also, reduce the height as well.
- Also, add a border to the last link in the lower navigation bar.

_See images below._

**Original** | **Your project**
  ----------- | -----------
  image | image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+      <div class="intro">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Accus antium
+          ipsam iste itaque nihil architecto, volu tatem sed facilis porro
+          maiores, rerum vitae.
+        </p>
+      </div>
+      <p class="date">05(FRI)-07(SUN) DEC 2021</p>
+      <p class="at">@ YouTube: Microverse Official Channel</p>
+    </header>
```
Comment (Temmarie) [inline index.html]: - [ ] As I previously stated, kindly add the `header` tag as a container for the navigation bar as this helps with semantic correctness and overall web page accessibility. This means the headline should be grouped with a `section` and then added to the `main` tag.
- [ ] As this is a headline, kindly use a `section` tag to group it. 
- [ ] The `Hello Micronauts` is a subheading and should be grouped using a `hx` tag instead of a `p` tag.
- [ ] Also, increase the font size of the headline intro to improve the readability and overall webpage aesthetics.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+            Trippy visuals from consectetur adipisicing elit. Repudiandae harum
+            nisi.
+          </p>
+        </div>
+        <div class="program-item">
+          <img src="assets/img/icon3.png" alt="" />
+          <h3>Interviews</h3>
+          <p>
+            On stage with amet consectetur. Repudiandae harum nisi, aliquid
+            adipisicing elit.
```
Comment (Temmarie) [inline index.html]: - [ ] You have grouped the main program using a `main` tag, this shouldn't be so, as the `main` HTML element represents the dominant content of the `body` of a document. So instead of doing this, It's advisable to group the entire content of the page, ( except the header which contains the nav and the footer) using a `main` tag, and then the individual groups like `main program`, `features speakers` will be grouped using a `section` tag. 
Note that, while the usage of semantic HTML tags helps improve page structuring, reinforces the meaning of information in web pages, it is also important to be semantically correct when using them.
- [ ] Reduce the spacings between the articles in the main program.

File: js/script.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+      'Big band format sit amet consectetur adipisicing elit. Repudiandae harum nisi.',
+  },
+  {
+    pic: 'assets/img/artist3.png',
+    name: 'James Git',
+    music: 'Punk Rock',
+    intro:
+      'For the first time in America amet consectetur elit. Repudiandae adipisicing.',
+  },
+  {
```
Comment (Temmarie) [inline js/script.js:55]: - [ ] Good job dynamically creating and rendering the featured speakers section(I especially like the names of the speakers 😉). It works perfectly well, you only have to make a few tweaks and you're good to go.
- The image shadow in the top left corner should be grey and white as shown in the images below.
- Reduce the font size of the italicised titles of the speakers.
- The subtitle should have a grey horizontal line beneath it, kindly add this.

_See images below_

**Original** | **Your project**
  ----------- | -----------
  image | image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+            ><img src="assets/img/logo.png" alt=""
+          /></a>
+          <a href="about.html">About</a>
+          <a href="#media">Media</a>
+          <a href="#lineup">Line up</a>
+          <a href="#buytickets">Buy tickets</a>
+          <div class="menu-close">
+            <img src="assets/img/Union.svg" alt="" />
+          </div>
+        </div>
```
Comment (Temmarie) [inline about.html]: - [ ] Kindly check my comments about using semantic HTML tags to structure the page.

File: about.html [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+          sunt.
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Accus antium
+          ipsam iste itaque nihil architecto, volu tatem sed consectetur
+          adipisicing elit facilis porro. Inventore necessitatibus suscipit
+          distinctio quidem dolorem placeat! Ad autem nisi vel repellendus.
+          Iste!
+        </p>
+      </div>
```
Comment (Temmarie) [inline about.html]: - [ ] Since this section doesn't show up on the about page, there's no point in adding it to the HTML file. Kindly remove it.

File: about.html [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+        <img src="assets/img/logo.png" alt="" />
+      </div>
+      <div class="separator"></div>
+      <h2>See the past Microverse Concerts</h2>
+      <p class="subtitle about">
+        Take a look at the last two Microverse Concert in San Francisco and New
+        York.
+      </p>
+      <div class="concerts-wrap about">
+        <div class="concert1">
```
Comment (Temmarie) [inline about.html]: - [ ] Kindly check out my comments about semantic HTML tags above and how to use them.
- [ ] The header intro should be a dark grey and should a larger and bolder font. The same goes for the contact info, and the middle section(about the concert section).

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @smunozmo   ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 
- The project is completely responsive ✔️ 
- You have created your own content while still following the given design template ✔️ 
- You did a good job using text transformations and inversions to change the colour of your logo ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:

- - -

### **Required Changes** ♻️
- [ ] Good job using semantic HTML to structure your webpage. There are a few issues that I have highlighted in the comments below. You can check [here](http://www.springboardseo.com/resources/what-is/semantic-html.html#:~:text=Semantic%20HTML%20is%20the%20correct,and%20context%20of%20web%20content.) for better instructions on how to properly use them.

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**
__Kindly make the necessary changes and submit another code review request.__

File: about.html (hunk 9 above)
Comment (smunozmo) [reply about.html]: Thank you @temmarie 🙌

File: .vscode/settings.json (hunk 2 above)
Comment (mricanho) [reply .vscode/settings.json:3]: - [ ] Right now is in the cache of your repository and the file doesn't go away, [check this link](https://stackoverflow.com/questions/45400361/why-is-gitignore-not-ignoring-my-files) about how to get rid of the .vscode file

Comment (mricanho) [reply .vscode/settings.json]: - [ ] Right now is in the cache of your repository and the file doesn't go away, [check this link](https://stackoverflow.com/questions/45400361/why-is-gitignore-not-ignoring-my-files) about how to get rid of the .vscode file

File: README.md [hunk 10]
```diff
@@ -1,10 +1,10 @@
...
-> Welcome to my HTML + CSS + Javascript Capstone project, feel free to look around.
+> Welcome to my HTML + CSS + Javascript Capstone project, feel free to look around. For this project I created a concert for developers sponsored by Microverse.
 
-- Live version: [GitHub Page](https://github.com/smunozmo/HTML-basic-JavaScript-Capstone)
+- Live version: [GitHub Page](https://smunozmo.github.io/Module_1_Capstone/)
 
-![screenshot](./)
+![screenshot](./assets/img/screenshot.png)
 
 ## Built With
```
Comment (mricanho) [inline README.md:9]: - [ ] Add more information on how to run the project locally, keeping in mind that there might be non-technical users trying to run it.  - You need to make the change that the last reviewer request, go back and check the README template, there's a section "Getting Started", a professional repository is super important for future projects and jobs.

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @smunozmo, 

Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

almost there
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 10 above)
Comment (smunozmo) [reply README.md:9]: Thank you @mricanho 🙌

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @smunozmo 😃, 

Great work and effort you made on the requested changes from the previous reviewers.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by smunozmo** [comment]
Thank you guys! @Temmarie @mricanho @MahmoudBakr23 ⚡️
